tests/integration/test_cli_benchmark_analytics_undo_display_utils.py

```
@@ -0,0 +1,2619 @@
+    def test_nonexistent_directory_raises_exit(self, tmp_path: Path) -> None:
+        import typer
+
+        from file_organizer.cli.utilities import _validate_search_params
+
+        with pytest.raises(typer.Exit):
+            _validate_search_params(10, tmp_path / "gone", None)
+
+    def test_limit_zero_returns_should_exit_true(self, tmp_path: Path) -> None:
+        from file_organizer.cli.utilities import _validate_search_params
+
+        resolved, should_exit = _validate_search_params(0, tmp_path, None)
```

Ruff (0.15.9)

[warning] Unpacked variable `resolved` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)